operations/security_and_authentication/Change_SMNP_credentials_on_Leaf_switches.md

```diff
@@ -0,0 +1,112 @@
+# Change SMNP credentials on Leaf switches
+This procedure changes the SNMP credentials on management leaf switches in the system. Either a single leaf switch can be updated to use new SNMP credentials, or update all leaf switches in the system to use the same global SNMP credentials.
+
+**Note**: This procedure will not update the default SNMP credentials used when new leaf switches are added to the system. To update the default SNMP credentials for new hardware follow the [Update default Air-Cooled BMC and Leaf switch SNMP credentials](Update_default_Air-Cooled_BMC_and_Leaf_switch_SNMP_credentials.md) procedure.
+
+## Prerequisites
+-   The Cray command line interface \(CLI\) tool is initialized and configured on the system.
+
+## Procedure
+
+1. List Leaf switches in system:
+    ```bash
+    ncn-m001# cray sls search hardware list --type comptype_mgmt_switch --format json | 
+        jq -r '["Xname", "Brand", "Alias"], (.[] | [.Xname, .ExtraProperties.Brand, .ExtraProperties.Aliases[0]]) | @tsv' | column -t
+    ```
+
+    Sample output for a system with 1 Aruba leaf switch:
+    ```
+    Xname       Brand  Alias
+    x3000c0w14  Aruba  sw-leaf-001
+    ```
+
+    Sample output for a system with 2 Dell leaf switches:
+    ```
+    Xname       Brand  Alias
+    x3000c0w14  Dell   sw-leaf-002
+    x3000c0w13  Dell   sw-leaf-001
+    ```
+    
+2. Update SNMP credentials for the `testuser` user on each leaf switch in the system. The SNMP `testuser` user requires 2 password to be provided for the SNMP Authentication and Privacy protocol passwords. Both of these passwords must be 8 characters or longer. In the examples below `foobar01` is the new SNMP Authentication password, and `foobar02` is the new SNMP Privacy password. 
+
+    1.  Aruba leaf switch configuration:
```

Comment:
Change to "Configure the Aruba leaf switch:"

Reply:
Should be resolved fixed now.